### PR TITLE
Interfaces - Word Smithing

### PR DESCRIPTION
**Pull request by NOYB**
Word smithing to remove a bunch of personalization ("You").

### CODE REVIEW

File: src/usr/local/www/interfaces.php [hunk 1]
```diff
@@ -354,6 +354,7 @@ function remove_bad_chars($string) {
 
 $pconfig['blockpriv'] = isset($wancfg['blockpriv']);
 $pconfig['blockbogons'] = isset($wancfg['blockbogons']);
+$pconfig['blocklin'] = isset($wancfg['blocklin']);
```
Comment (phil-davis) [inline src/usr/local/www/interfaces.php]: Is this fixing a bug? I don't see 'blocklin' anywhere else in this code.

File: src/usr/local/www/interfaces.php [hunk 2]
```diff
@@ -622,7 +622,7 @@ function remove_bad_chars($string) {
 			break;
 		case "dhcp6":
 			if (in_array($wancfg['ipaddrv6'], array())) {
-				$input_errors[] = sprintf(gettext("You have to reassign the interface to be able to configure as %s."), $_POST['type6']);
+				$input_errors[] = sprintf(gettext("The interface has to be reassigned to be able to configure as %s."), $_POST['type6']);
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: Too many instances of "to"  there, and in several places in this commit, should probably be:
"The interface must be reassigned to configure as %s"

File: src/usr/local/www/interfaces.php [hunk 3]
```diff
@@ -632,7 +632,7 @@ function remove_bad_chars($string) {
 			foreach ($ifdescrs as $ifent => $ifdescr) {
 				if ($if != $ifent && ($config[interfaces][$ifent]['ipaddrv6'] == $_POST['type6'])) {
 					if ($config[interfaces][$ifent]['prefix-6rd'] == $_POST['prefix-6rd']) {
-						$input_errors[] = gettext("You can only have one interface configured in 6rd with same prefix.");
+						$input_errors[] = gettext("Only one interface can be configured in 6rd with same prefix.");
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: Seems awkward... Perhaps "Only one interface can be configured within a single 6rd prefix" or "An interface already exists using that 6rd prefix" (which implies you cannot have a second)

File: src/usr/local/www/interfaces.php [hunk 4]
```diff
@@ -641,36 +641,36 @@ function remove_bad_chars($string) {
...
 			}
 
 			if ($_POST['track6-prefix-id--hex'] != "" && !is_numeric("0x" . $_POST['track6-prefix-id--hex'])) {
-				$input_errors[] = gettext("You must enter a valid hexadecimal number for the IPv6 prefix ID.");
+				$input_errors[] = gettext("A valid hexadecimal number must be entered for the IPv6 prefix ID.");
 			} else {
 				$track6_prefix_id = intval($_POST['track6-prefix-id--hex'], 16);
 				if ($track6_prefix_id < 0 || $track6_prefix_id > $_POST['ipv6-num-prefix-ids-' . $_POST['track6-interface']]) {
-					$input_errors[] = gettext("You specified an IPv6 prefix ID that is out of range.") .
+					$input_errors[] = gettext("An IPv6 prefix ID that is out of range was specified.") .
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: "The specified IPv6 Prefix ID is out of range"

File: src/usr/local/www/interfaces.php [hunk 5]
```diff
@@ -1620,7 +1620,7 @@ function check_wireless_mode() {
 		$old_wireless_mode = $wancfg['wireless']['mode'];
 		$wancfg['wireless']['mode'] = $_POST['mode'];
 		if (!interface_wireless_clone("{$wlanif}_", $wancfg)) {
-			$input_errors[] = sprintf(gettext("Unable to change mode to %s.	 You may already have the maximum number of wireless clones supported in this mode."), $wlan_modes[$wancfg['wireless']['mode']]);
+			$input_errors[] = sprintf(gettext("Unable to change mode to %s.	 May already have the maximum number of wireless clones supported in this mode."), $wlan_modes[$wancfg['wireless']['mode']]);
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: "The maximum number of wireless clones supported in this mode may have been reached."

File: src/usr/local/www/interfaces.php [hunk 6]
```diff
@@ -1702,7 +1702,7 @@ function build_gatewayv6_list() {
 
 if (is_subsystem_dirty('interfaces')) {
 	print_apply_box(sprintf(gettext("The %s configuration has been changed."), $wancfg['descr']) . "<br />" .
-					gettext("You must apply the changes in order for them to take effect.") . "<br />" .
+					gettext("The changes must be applied in order for them to take effect.") . "<br />" .
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: Still seems wordy... "The changes must be applied to take effect" gets the point across, or even "The changes must be applied"

File: src/usr/local/www/interfaces.php [hunk 7]
```diff
@@ -3377,7 +3377,7 @@ function report_failure(request, textStatus, errorThrown) {
 		if (textStatus === "error" && request.getResponseHeader("Content-Type") === "text/plain") {
 			alert(request.responseText);
 		} else {
-			alert("Sorry, we could not create your IPv4 gateway at this time.");
+			alert("Could not create the IPv4 gateway at this time.");
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: Seems awkward/wordy. Perhaps "The IPv4 gateway could not be created"

File: src/usr/local/www/interfaces.php [hunk 8]
```diff
@@ -3426,7 +3426,7 @@ function report_failure_v6(request, textStatus, errorThrown) {
 		if (textStatus === "error" && request.getResponseHeader("Content-Type") === "text/plain") {
 			alert(request.responseText);
 		} else {
-			alert("Sorry, we could not create your IPv6 gateway at this time.");
+			alert("Could not create the IPv6 gateway at this time.");
```
Comment (jim-p) [inline src/usr/local/www/interfaces.php]: See above

File: src/usr/local/www/interfaces.php (hunk 3 above)
Comment (gonzopancho) [inline src/usr/local/www/interfaces.php]: s/can/may/

**Comment by NOYB** [comment]
Thanks for the additional updates Jim.  Much better.  Thank you.

**Pull request merged**